pages/assessment.tsx

```diff
@@ -167,6 +165,16 @@ const Assessment: NextPage = () => {
     }
   };
 
+  const findAndSetCurrentPage = (assessments: AssessmentType[]) => {
```

Comment:
I dont think you necessarily need to take in assessments? Or maybe you need it for the initial run?

Reply:
If we want to use it in `handleSetMaxItems()` we have to take in assessments as the initial run uses  assessments/answers from `getAssessmentData()` while `handleSetMaxItems()` uses `assessments from the state`